### DISCUSSION TITLE
Jupyter

### DISCUSSION THREAD

**Discussion by Peio47** [robot-soccer-kit/robot-soccer-kit#55] [Q&A] [answered]
Bonjour,
J'ai essayé de lancer un programme client depuis une page Jupyter et le script s'est arrêté sans aucun effet.
Y a-t-il une manipulation à faire ?
En vous remerciant

**Comment by cgaspard3333** [comment, marked as answer]
Bonjour, 
Normalement si Jupyter utilise bien la même version de Python que celle que vous utilisez habituellement (avec le module RSK d'installé), il ne devrait pas y avoir de problème. 
Je suis dispo pour vous assister à distance avec TeamViewer, vous pouvez m'appeler au numéro de téléphone en bas de mon email si vous êtes disponibles.
